media-tv/tvbrowser-bin/tvbrowser-bin-4.0.1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit desktop
+
+DESCRIPTION="Themeable and easy to use TV Guide - written in Java"
+HOMEPAGE="https://www.tvbrowser.org"
+MY_PN="tvbrowser"
+MY_P="${MY_PN}-${PV}"
+SRC_URI="mirror://sourceforge/project/tvbrowser/TV-Browser%20Releases%20%28Java%208%20and%20higher%29/${PV}/${MY_PN}_${PV}_bin.tar.gz"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="virtual/jdk:1.8"
+
+S="${WORKDIR}"/"${MY_P}"
+
+src_install() {
+	# Copy files and directories
+	dodir /opt/"${P}"
+	insinto /opt/"${P}"
+	doins -r *
+
+	# Generate launcher
+	local tvbrowser_bin="tvbrowser"
+	exeinto /usr/bin
+	newexe - "${tvbrowser_bin}" <<-_EOF_
+#!/bin/sh
```

Comment:
Remember indentation, and I believe bash is preferred with launchers / scripts like these. (I think sh is a symlink to system default shell interpreter, so go with bash)

Reply:
Fixed indentation; I thought it would be necessary for the resulting file to have no indentation, but it luckily works without.
As for the png file, I don't think I'd have to do anything about it as a maintainer other than report it upstream.
I've also fixed the shell because we reference /bin/bash later anyway. I've looked it up in other ebuilds and they also use /bin/bash, and since portage depends on /bin/bash too it should be safe.

Reply:
Also removed the tvbrowser_bin local variable.